<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>29</y>
     <width>380</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Betriebsart</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="radioButton_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Ganznacht</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QRadioButton" name="radioButton_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Abschaltung spätnachts</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QRadioButton" name="radioButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::PreventContextMenu</enum>
      </property>
      <property name="text">
       <string>Dimmung spätnachts</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Uhrzeit Halbnacht/Dim ein</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>01:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>00:30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>00:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23:30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22:30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21:30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20:30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20:00</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Uhrzeit Halbnacht/Dim aus</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="comboBox_2">
      <item>
       <property name="text">
        <string>00:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>00:30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>01:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>01:30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>02:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>02:30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>03:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>03:30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>04:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>04:30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>05:00</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>05:30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>06:30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>07:00</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Dimmniveau in %</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="spinBox">
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="specialValueText">
       <string/>
      </property>
      <property name="suffix">
       <string>%</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Eingeschaltet in Stunden/Jahr (100%)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>Auschaltdauer/gedimmt in Stunden/Jahr</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="label_8">
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>Jahresbrenndauer gesamt</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="label_10">
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>4100</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Berechnen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_20">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
